sphinx/ext/autodoc/preserve_defaults.py

```diff
@@ -0,0 +1,82 @@
+"""
+    sphinx.ext.autodoc.preserve_defaults
```

Comment:
I'm still debating the name of this extension. I'm looking for a better name for this...